### ISSUE TITLE
Clean up public interfaces of stevia modules - especially the ast module

### ISSUE THREAD

**Issue by Robbepop** [Robbepop/stevia#8]
Currently stevia was implemented without proper adjustment on the outer interfaces to the module. The ast module for example has a giantic interface which should and certainly could be minimized. To allow for later decoupling of modules this should be done beforehand.

**Comment by Robbepop** [comment]
Duplicate of https://github.com/Robbepop/stevia/issues/17 (lol).
Close this since the other is more informative.
